content/tutorials/manage-the-rippled-server/configuration/configure-history-sharding.md

```diff
@@ -6,36 +6,41 @@
 
 To configure your `rippled` to store shards of ledger history, complete the following steps:
 
-## 1. Determine how much space to allocate for the shard store
+## 1. Determine how much many shards to maintain
```

Comment:
"Determine how much many shards to maintain" -> "Determine how many shards to maintain"